scripts/PerfHarness/run.bat

```diff
@@ -1 +1 @@
-..\..\dotnetcli\dotnet.exe run -c Release -- --perf:outputdir results --perf:typenames %1
+..\..\dotnetcli\dotnet.exe run -c Release -- --perf:outputdir results --perf:typenames %1 --perf:collect default+CacheMisses+InstructionRetired
```

Comment:
Do u care about allocations? If so, consider adding gcapi as well.

Reply:
I was thinking about it, but many of our APIs simply don't allocate and so this stat shows zero